plugins/utils/todo-comments.nix

```diff
@@ -431,14 +432,13 @@ lib.nixvim.neovim-plugin.mkNeovimPlugin config {
     extraPackages = [ cfg.ripgrepPackage ];
 
     keymaps = lib.pipe cfg.keymaps [
-      (filterAttrs (n: v: v != null))
+      (filterAttrs (n: keymap: keymap != null))
       (mapAttrsToList (
         name: keymap: {
-          inherit (keymap) key options;
-          mode = "n";
+          inherit (keymap) key mode options;
           action =
             let
-              cmd = keymapsActions.${name};
+              cmd = keymap.action;
```

Comment:
tbh this is probably more readable if we inline this into the action string below?

```suggestion
```

and then

```nix
"<cmd>${keymap.action}${cwd}${keywords}<cr>";
```